src/ui/src/builder/BuilderJournal.vue

```diff
@@ -299,30 +395,42 @@ function handleReRun(entry: JournalEntry) {
 	// Will trigger the blueprint/workflow execution with entry data
 }
 
-function handleGoToTrigger(entry: JournalEntry) {
-	// Close the drawer
-	selectedEntry.value = null;
+function handleGoToTrigger(entry: UnifiedEntry) {
+	if (entry.entryType !== "execution") return;
 
 	// Go to the trigger component
 	goToComponentParentPage(entry.trigger.component.id);
-	selectChild(entry.trigger.component.id);
-	pushToast({
-		type: "success",
-		message: `Jumped to ${entry.title}`,
-	});
+	if (selectChild(entry.trigger.component.id)) {
+		pushToast({
+			type: "success",
+			message: `Jumped to ${entry.title}`,
+		});
```

Comment:
Do we need a success confirmation, I mean the user will see that it worked, no ?